src/main/io/serial_resource.c

```diff
@@ -60,9 +60,8 @@ serialType_e serialType(serialPortIdentifier_e identifier)
     }
 #endif
 #ifdef USE_SOFTSERIAL
-    if (identifier >= SERIAL_PORT_SOFTSERIAL_FIRST
-        && identifier < SERIAL_PORT_SOFTSERIAL_FIRST + SERIAL_SOFTSERIAL_MAX) {
-        // sotserials always start from first index, without holes
+    if (identifier >= SERIAL_PORT_SOFTSERIAL1 && identifier < SERIAL_PORT_SOFTSERIAL1 + SERIAL_SOFTSERIAL_MAX) {
```

Comment:
I'd keep it consistent with other ports, using SERIAL_PORT_SOFTSERIAL_FIRST

Reply:
Yes. Edits required to take advantage of your PR